### ISSUE TITLE
One_shot option std out should be raw json

### ISSUE THREAD

**Issue by mrichins** [google/mtail#312]
When building an mtail application, I enjoy using the `-one_shot` option to test the correctness of my mtail progs. However, the output looks like json, but includes a little string "Metrics store:" before the rest of the metrics json is written. This makes programmatically plugging into mtail to automate testing a bit harder.

I had to work around this by invoking it this way `mtail -one_shot -progs ./progs -logs <test_log> | tail -c +15  | jq` 

Basically, I see no benefit to this string existing in the output, but it could also be another  CLI option. More than happy to write the PR myself too.

Thanks! Mason
